src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Diagnostics/DocumentPullDiagnosticsEndpointTest.cs

```diff
@@ -40,9 +42,11 @@ public void ApplyCapabilities_AddsExpectedCapabilities()
         Assert.NotNull(serverCapabilities);
         Assert.NotNull(serverCapabilities.DiagnosticProvider);
         Assert.NotNull(serverCapabilities.DiagnosticProvider.DiagnosticKinds);
-        Assert.Single(serverCapabilities.DiagnosticProvider.DiagnosticKinds);
+        Assert.Equal(2, serverCapabilities.DiagnosticProvider.DiagnosticKinds.Length);
```

Comment:
FWIW, I don't think this assert is strictly necessary. I'm pretty sure that `Assert.Collection` will fail if the length doesn't match the number of item assertions.